### PR TITLE
chore: generate schema-minimal as part of generate

### LINKED ISSUE #4811: Workflow failure: minimal-schema
## Workflow Failure

[minimal-schema](https://github.com/pulumi/pulumi-aws/blob/master/.github/workflows/aws-minimal-schema.yml) has failed. See the list of failures below:

- [2024-11-21T00:58:34.000Z](https://github.com/pulumi/pulumi-aws/actions/runs/11944515628)

### PR DESCRIPTION
**Pull request by corymhall**
re #4822, closes #4811

### CODE REVIEW

File: provider/cmd/pulumi-resource-aws/schema-minimal.json [hunk 1]
```diff
@@ -278857,7 +278857,7 @@
...
...xclusiveArgs{\n\t\t\tRoleName:   pulumi.Any(exampleAwsIamRole.Name),\n\t\t\tPolicyArns: pulumi.StringArray{},\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\treturn nil\n\t})\n}\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.aws.iam.RolePolicyAttachmentsExclusive;\nimport com.pulumi.aws.iam.RolePolicyAttachmentsExclusiveArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        var example = new RolePolicyAttachmentsExclusive(\"example\", RolePolicyAttachmentsExclusiveArgs.builder()\n            .roleName(exampleAwsIamRole.name())\n            .policyArns()\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  example:\n    type: aws:iam:RolePolicyAttachmentsExclusive\n    properties:\n      roleName: ${exampleAwsIamRole.name}\n      policyArns: []\n```\n\u003c!--End PulumiCodeChooser --\u003e\n",
```
Comment (flostadler) [inline provider/cmd/pulumi-resource-aws/schema-minimal.json:278860]: Wait, did we end up comitting the wrong minimal schema previously?

**Review by flostadler — APPROVED** [review]

**Comment by t0yv0** [comment]
We should also remove the machinery made redundant by this.

**Review by t0yv0 — APPROVED** [review]

File: provider/cmd/pulumi-resource-aws/schema-minimal.json (hunk 1 above)
Comment (corymhall) [reply provider/cmd/pulumi-resource-aws/schema-minimal.json:278860]: Yes, the minimal schema check was not a required check so a PR was merged without it.

Comment (t0yv0) [reply provider/cmd/pulumi-resource-aws/schema-minimal.json:278860]: Yeah I've missed that looks like. IN cases like this also feel free to revert and send it back to me.

**Comment by corymhall** [comment]
This doesn't work, need to revisit next week.

**Comment by t0yv0** [comment]
I'll pick this up.

**Pull request closed without merging**